<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoremIpsumClass</class>
 <widget class="QMainWindow" name="LoremIpsumClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>624</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Lorem Ipsum Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>561</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="genBtn">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyBtn">
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearBtn">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>571</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Sentences</string>
    </property>
    <widget class="QCheckBox" name="stdCB">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Use classic Lorem Ipsum paragraph</string>
     </property>
    </widget>
    <widget class="QLabel" name="minWdLB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Minimum # of words:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="minLE">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>3</string>
     </property>
    </widget>
    <widget class="QLabel" name="maxWdLB">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Maximum # of words:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="maxLE">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>10</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>571</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Paragraphs</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string># of paragraphs:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="numParaLE">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Minimum # of sentences:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="minSentLE">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Maximum # of sentences:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="maxSentLE">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>60</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="resultTB">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>571</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="loremipsum.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>LoremIpsumClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
